### ISSUE TITLE
Black screen after selecting install macos.

### ISSUE THREAD

**Issue by Dave-Swagten** [yusufklncc/Lenovo-Legion-5-Hackintosh#7]
Hey there, I've tried using this efi but I cant seem to install macOS.. after I select install macOS I see a lot of logs and a black screen appears. The screen is not off, but just black. normally, the apple icon should show up but nothing happens. I use a 
Legion 5 15IMH05H.

Thanks for reading!

**Comment by yusufklncc** [comment]
Can you try with new release?

**Comment by jemmno** [comment]
Same here, black screen

**Comment by Dave-Swagten** [comment]
I fixed it by going into bios and changing dynamic graphics to discrete graphics. However, the hackintosh will run super duper slow with alot of stutters because you are running it off your nvida card. The screen is also 20hz and you cant change it. This is far from ideal 😅. Im trying to find out how to fix this, but ive had no luck so far. I tried disabling the gpu but now it only shows the logs. I might have to reinstall everything and try again.
